app/views/bulk_updates_searches/new.html.erb

```diff
@@ -31,36 +31,55 @@
     <% end %>
 
     <% if params[:query] %>
-      <% if @collections.any? %>
-        <p><%= page_entries_info @collections, entry_name: 'Collections' %></p>
-      <% end %>
+      <%= form_tag(new_bulk_updates_path, method: :post) do %>
+
+        <% if @collections.any? %>
+          <p><%= page_entries_info @collections, entry_name: 'Collections' %></p>
+        <% end %>
+
+        <table id="bulk-updates-search-results">
+          <thead>
+            <tr>
+              <th class="align-c"><input type="checkbox" name="checkall" class="checkall" data-group="bulk_update_collections[]" /></th>
+              <th>Collection</th>
+              <th>Short Name</th>
+              <th>Version</th>
+            </tr>
+          </thead>
+          <tbody>
+            <% if @collections.any? %>
+            <% @collections.each do |collection| %>
```

Comment:
indentation within the `if` erb tags. or we can just make the edit while doing 863.